docs/source/hotregion.rst

```diff
@@ -16,6 +30,7 @@ intense regions of the source photosphere.
               cede, concentric
     :show-inheritance:
 
+
 .. autoclass:: xpsi.HotRegion.RayError
     :show-inheritance:
```

Comment:
could you add the :show-inheritance: command to the class that inherit from ParameterSubspace but is not shown in the docs, like you pointed out, please?

Reply:
Sure :) let me know if I should modify something else

Reply:
I think I have done that too now, but let me know if something is still missing